frontend/javascripts/oxalis/view/right-border-tabs/comment_tab/comment_tab_view.tsx

```diff
@@ -100,7 +100,7 @@ function CommentTabView(props: Props) {
 
   const [isSortedAscending, setIsSortedAscending] = useState(true);
   const [sortBy, setSortBy] = useState(SortByEnum.NAME);
-  const [collapsedTreeIds, setCollapsedTreeIds] = useState<React.Key[]>([]);
+  const [expandedTreeIds, setExpandedTreeIds] = useState<React.Key[]>([]);
```

Comment:
Just a rename of the state variable since the `antd` `<Tree/>`-component uses the wording expanded nodes etc.